frontend/src/components/SearchInputGroup.vue

```diff
@@ -9,14 +9,22 @@
           </div>
 
           <ComboboxInput v-if="selectedItem == null" v-focus class="w-full h-10 rounded-l-md border border-gray-300 bg-white py-2 px-10 shadow-xs focus:border-primary focus:outline-hidden focus:ring-1 focus:ring-primary sm:text-sm disabled:bg-primary-l2" placeholder="John Doe" @change="query = $event.target.value"/>
-          <input v-else v-model="selectedItem.name" class="w-full h-10 rounded-l-md border border-gray-300 bg-primary-l2 py-2 px-10 shadow-xs focus:border-primary focus:outline-hidden focus:ring-1 focus:ring-primary sm:text-sm" readonly />
+          <div v-else class="w-full h-10 rounded-l-md border border-gray-300 bg-primary-l2 py-2 px-10 flex items-center justify-between shadow-xs sm:text-sm">
+            <span class="truncate">{{ selectedItem.name }}</span>
+            <span v-if="selectedItem.type === 'GROUP'" class="text-gray-500 text-xs italic mr-6">
+              {{ selectedItem.memberSize }} {{ t('vaultDetails.sharedWith.members') }}
```

Comment:
This is not `vaultDetails` and also use the placeholder approach (`{0} Members`)